src/providers/steem/dsteem.js

```diff
@@ -735,9 +731,8 @@ export const postContent = async (
 // Re-blog
 export const reblog = async (account, pinCode, author, permlink) => {
   const pin = getDigitPinCode(pinCode);
-
-  if (account.local.authType === AUTH_TYPE.MASTER_KEY) {
-    const key = decryptKey(account.local.postingKey, pin);
+  const key = getAnyPrivateKey(account.local, pin);
+  if (account.local.authType !== AUTH_TYPE.STEEM_CONNECT && key) {
```

Comment:
what if key doesn't exist this should be return something
```
if(!key){
return null;
}
```
or something like that. instead both check also it can return

Reply:
@mistikk